extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js

```diff
@@ -301,6 +301,11 @@ class LifecycleBucketProcessor {
                     return cb();
                 }
 
+                if (err.code === 'NoSuchBucket') {
+                    this._log.error('skipping non-existent bucket', { bucket });
```

Comment:
Should this be `info` unless you see a reason to dump the logs when it reaches this code path.